File: baekjoon/bronze/level1/test_1373/java/JunitTest.java
package bronze.level1.test_1373.java;

import org.junit.Test;
import util.TestUtil;

import java.io.IOException;

import static org.junit.Assert.assertEquals;
import static util.ThrowingRunnable.runUnchecked;

public class JunitTest {

    @Test
    public void test_1() throws IOException {
        String input = "11001100";
        String expectedOutput = "314";
        
        String actualOutput = TestUtil.executeTest(input, () -> runUnchecked(() -> Main.main(new String[0])));
        assertEquals(expectedOutput, actualOutput);
    }

}

File: baekjoon/bronze/level1/test_2775/kotlin/JunitTest.kt
package bronze.level1.test_2775.kotlin;

import junit.framework.TestCase.assertEquals
import org.junit.Test
import util.TestUtil

class JunitTest {

    @Test
    fun test1() {
        val input = """
            2
            1
            3
            2
            3
        """.trimIndent()
        val expectedOutput = """
            6
            10
            
        """.trimIndent()

        val actualOutput = TestUtil.executeTest(input) { main() }
        assertEquals(expectedOutput, actualOutput)
    }

}
